recipes/libspatialite/all/conanfile.py

```diff
@@ -85,28 +85,29 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("sqlite3/3.42.0")
-        self.requires("zlib/1.2.13")
+        # sqlite.h is included by spatialite/sqlite.h public header
+        self.requires("sqlite3/3.42.0", transitive_headers=True)
```

Comment:
```suggestion
        self.requires("sqlite3/3.42.2", transitive_headers=True)
```
Fixes conflict with proj